pkg/ingester/ingester.go

```diff
@@ -206,6 +206,9 @@ type Config struct {
 	UpdateIngesterOwnedSeries       bool          `yaml:"track_ingester_owned_series" category:"experimental"`
 	OwnedSeriesUpdateInterval       time.Duration `yaml:"owned_series_update_interval" category:"experimental"`
 
+	// TODO Make "false" by default, after IngestStorageConfig.Enabled is stable
+	EnablePushAPI bool `yaml:"enable_push_api" category:"experimental"`
```

Comment:
So far we're keeping hidden all the ingest storage config.

```suggestion
	EnablePushAPI bool `yaml:"enable_push_api" category:"experimental" doc:"hidden"`
```